content/docs/hooks-state.md

```diff
@@ -1,38 +1,39 @@
 ---
 id: hooks-state
-title: Using the State Hook
+title: Holat hukini ishlatish
 permalink: docs/hooks-state.html
 next: hooks-effect.html
 prev: hooks-overview.html
 ---
 
-*Hooks* are a new addition in React 16.8. They let you use state and other React features without writing a class.
+*Huklar* - React 16.8'ga kiritilgan yangilik. Ular sizga klass yozmasdan turib holat (state) va React'ning boshqa qulayliklarini ishlatishga yordam beradi.
 
-The [introduction page](/docs/hooks-intro.html) used this example to get familiar with Hooks:
+Sizni huklar bilan tanishtirish uchun [kirish sahifasida](/docs/hooks-intro.html) quyidagi misolni keltirgan edik:
 
 ```js{4-5}
 import React, { useState } from 'react';
 
 function Example() {
-  // Declare a new state variable, which we'll call "count"
+  // "count" deb nomlangan, yangi holat (state) oʻzgaruvchisini yaratish
   const [count, setCount] = useState(0);
 
   return (
     <div>
-      <p>You clicked {count} times</p>
+      <p>Siz {count} marta bosdingiz</p>
       <button onClick={() => setCount(count + 1)}>
-        Click me
+        Meni bosing
       </button>
     </div>
   );
 }
 ```
 
-We'll start learning about Hooks by comparing this code to an equivalent class example.
+Biz huklar haqida oʻrganishni ularni klassdagi akslari bilan taqqoslagan holda boshlaymiz.
 
-## Equivalent Class Example {#equivalent-class-example}
+<!-- ## Equivalent Class Example {#equivalent-class-example} -->
```

Comment:
Comment?

Reply:
Couldn't translate the "Equivalent" word, so left the original text as comment